payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/TransformSpecToElements.kt

```diff
@@ -57,6 +58,8 @@ class TransformSpecToElements(
                 is StaticTextSpec -> it.transform(merchantName)
                 is AfterpayClearpayTextSpec ->
                     it.transform(requireNotNull(amount))
+                is AffirmTextSpec ->
+                    it.transform(requireNotNull(amount))
```

Comment:
We don't need the amount just yet.

Reply:
Good point! There were a few spots I had that floating around. They should all be removed now.